holoviews/plotting/renderer.py

```diff
@@ -358,7 +357,26 @@ def embed_assets(cls):
                              if f is not None )
         widgetcss = '\n'.join(open(find_file(path, f), 'r').read()
                               for f in css if f is not None)
-        return widgetjs, widgetcss
+
+        dependencies = {}
+        if core:
+            dependencies.update(cls.core_dependencies)
+        if extras:
+            dependencies.update(cls.extra_dependencies)
+        for backend in backends:
+            dependencies['backend'] = Store.renderers[backend].backend_dependencies
+
+        js_html, css_html = '', ''
+        for _, dep in sorted(dependencies.items(), key=lambda x: x[0]):
+            for js in dep.get('js', []):
+                js_html += '\n<script src="%s" type="text/javascript"></script>' % js
+            for css in dep.get('css', []):
+                css_html += '\n<link rel="stylesheet" href="%s">' % css
+
+        js_html += '\n<script type="text/javascript">%s</script>' % widgetjs
+        css_html += '\n<style>%s</style>' % widgetcss
+
+        return js_html, css_html
```

Comment:
Looks good. If necessary, could one day be made to spit out a dictionary of resource locations if we ever needed them without the surrounding HTML tags.